app/public/index.html

```diff
@@ -6,13 +6,18 @@
     <title>Nazareth Nuggets</title>
     <link rel="stylesheet" href="./css/normalize.min.css">
     <link rel="stylesheet" href="./css/main.css">
+    <script src="https://use.fontawesome.com/c487c1de1c.js"></script>
+    <link href="https://fonts.googleapis.com/css?family=Comfortaa" rel="stylesheet">
     <link rel="stylesheet" href="https://unpkg.com/leaflet@1.0.3/dist/leaflet.css"
    integrity="sha512-07I2e+7D8p6he1SIM+1twR5TIrhUQn9+I6yjqD53JQjFiMf8EtC93ty0/5vJTZGF8aAocvHYNEDJajGdNx1IsQ=="
    crossorigin=""/>
   </head>
   <body>
     <!-- HEADER -->
-
+    <div class="header">
```

Comment:
it is kinda the same the im changing it though so that it would be more obvious that it is a header